### PR TITLE
added upload abort/cancel support

### PR DESCRIPTION
**Pull request by marekszczepansky**
#195 This should add support to abort upload if already started.

### CODE REVIEW

**Comment by jkuri** [comment]
does this actually works?

**Comment by marekszczepansky** [comment]
Sure! I'm using this in my current project and this works OK.

**Comment by kkhalil1** [comment]
When this can be merged ?

File: src/classes/uploaded-file.class.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export class UploadedFile {
+  private _xhr: XMLHttpRequest | undefined;
```
Comment (jkuri) [inline src/classes/uploaded-file.class.ts]: can you remove `private` here please?

**Comment by jkuri** [comment]
thanks @marekszczepansky!

**Comment by marekszczepansky** [comment]
You're welcome :)

**Comment by kkhalil1** [comment]
Yay ! thank you both 👍

**Pull request merged**